pkg/config/finder.go

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright 2019 Chef Software, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package config
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	DefaultChefDirectory            = ".chef"
+	DefaultChefWorkstationDirectory = ".chef-workstation"
+	DefaultFileName                 = "config.toml"
+)
+
+// finds the configuration file (default .chef-workstation/config.toml) inside the current
+// directory and recursively, plus inside the $HOME directory
+func FindChefWorkstationConfigFile() (string, error) {
+	return FindConfigFile(DefaultFileName)
+}
+
+// @afiune do we want to find other files like knife.rb and config.rb?
```

Comment:
At this point, no. I think we could remove this comment and the commented out functions